cmake/dependencies/flamegpu2-visualiser.cmake

```diff
@@ -8,7 +8,7 @@ cmake_policy(SET CMP0079 NEW)
 
 # Set the visualiser repo and tag to use unless overridden by the user.
 # @todo - If the git version has changed in this file, fetch again? 
-set(DEFAULT_VISUALISATION_GIT_VERSION "flamegpu-2.0.0-alpha.1")
+set(DEFAULT_VISUALISATION_GIT_VERSION "d35ad8f265364ee05bca6c4bec62374d1dbbea94")
```

Comment:
Worth updating release guide to include visualisation release tag/cmake bind, if required?

Reply:
> The CMake version pinning of the visualisation repo is correct. Ideally to a tag specific to the main repo release (i.e. flamegpu-2.0.0-alpha)

It's mentioned but I'll clarify that a new tag should be made if required.

Reply:
Fair, I guess I don't recall (and maybe even missed it) when I did last release.